### ISSUE TITLE
Two or more jails with 'backend = systemd' fails : Error : can't start new thread

### ISSUE THREAD

**Issue by blitzter47** [fail2ban/fail2ban#1513]
### Environment:
- Fail2Ban version (including any possible distribution suffixes): v0.9.3
- OS, including release name/version: Debian GNU/Linux testing/unstable stretch/sid 3.10-3-686-pae SMP 3.10.11-1 (2013-09-10) i686
- [X] Fail2Ban installed via OS/distribution mechanisms
- [X] You have not applied any additional foreign patches to the codebase
- [X] Some customizations were done to the configuration (provide details below is so)

### The issue:

I have four jails (ready to be) enabled. Two of them use `backend = polling` (taking into account https://github.com/fail2ban/fail2ban/issues/959#issuecomment-74901200) and two others use `backend = systemd`.

My problem occurs when I try to enable two or more jails configured to `backend = systemd`. After I restart fail2ban-client (`sudo fail2ban-client stop && sudo fail2ban-client -x start`), it outputs :
````
ERROR  NOK: ("can't start new thread",)
ERROR  NOK: ("can't start new thread",)
ERROR  NOK: ("can't start new thread",)
ERROR  NOK: ("can't start new thread",)
````
and fail2ban doesn't block anything in any jail.

But with only one jail enabled to use `backend = systemd` (no matter how many jails using `polling` as backend are enabled), no error and everything work fine.

#### Steps to reproduce
* Configure [sshd] and [sendmail-reject] to use systemd as backend.
* Add `journalmatch = _SYSTEMD_UNIT=sendmail.service + _COMM=sm-mta` to [sendmail-reject]
* Enable both jails
* Restart Fail2ban

#### Any additional information

In `/etc/fail2ban/jail.local` file, under [default], switching between `backend = systemd` and `backend = auto` changes nothing to the problem, even if `backend = _systemd or polling_` is specified in each jail explicitely, and taking into account https://github.com/fail2ban/fail2ban/issues/959#issuecomment-74901200.

#### Relevant lines from monitored log files in question:
fail2ban.log
```
2016-08-16 15:03:24,366 fail2ban.transmitter    [24475]: DEBUG   Command: ['start', 'sshd']
2016-08-16 15:03:24,366 fail2ban.transmitter    [24475]: WARNING Command ['start', 'sshd'] has failed. Received RuntimeError("can't start new thread",)
2016-08-16 15:03:24,367 fail2ban.transmitter    [24475]: DEBUG   Command: ['start', 'sendmail-reject']
2016-08-16 15:03:24,367 fail2ban.transmitter    [24475]: WARNING Command ['start', 'sendmail-reject'] has failed. Received RuntimeError("can't start new thread",)
```

**Comment by sebres** [comment]
closed as duplicate of #969

**Comment by sebres** [comment]
Note please https://github.com/fail2ban/fail2ban/issues/969#issuecomment-240350774
